.github/workflows/ci.yml

```diff
@@ -670,20 +722,34 @@ jobs:
       # https://github.com/tokio-rs/tokio/issues/5373
       - name: Check
         # We use `--skip io-uring` since io-uring crate doesn't provide a binding for the i686 target.
-        run: cargo hack check -Zbuild-std --target target-specs/i686-unknown-linux-gnu.json -p tokio --feature-powerset --skip io-uring --depth 2 --keep-going
+        run: |
+          cargo hack check \
+            -Zbuild-std \
+            --target target-specs/i686-unknown-linux-gnu.json \
+            -p tokio \
+            --feature-powerset \
+            --skip io-uring \
+            --depth 2 \
+            --keep-going
         env:
-          RUSTFLAGS: --cfg tokio_unstable --cfg tokio_taskdump -Dwarnings
+          RUSTFLAGS: --cfg tokio_unstable -Dwarnings
 
   features:
-    name: features ${{ matrix.name }}
+    name: features exclude ${{ matrix.name }}
```

Comment:
This changed the name of this job, we need to update the repo setting to unblock the CI.

<img width="1562" height="596" src="https://github.com/user-attachments/assets/465fe7ab-346c-452f-b4eb-c2974951789a" />